ci/docker/openstack-cpi-release/Dockerfile

```
@@ -1,56 +1,29 @@
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+ENV BOSH_CLI_VERSION=7.10.10
+
+RUN apt-get -y update \
+    && apt-get install -y --no-install-recommends \
+        locales \
+    && locale-gen en_US.UTF-8 \
+    && update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+
+# ruby needed for BATS (bosh-acceptance-tests is an RSpec suite)
+RUN apt-get install -y --no-install-recommends \
```

Hadolint (2.15.1)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)